chapters/data-processing.tex

```diff
@@ -0,0 +1,783 @@
+After that, identifying information is typically no longer needed,
+the third section outlines steps to remove direct identifiers.
+Finally, the last section discusses how to explore each variable in your dataset and 
+make sure that the dataset is as well documented and easy to use as possible.
+
+
+\end{fullwidth}
+
+%------------------------------------------------
+
+\subsection{Before you get started}
+
+% Don't mess with the raw data
+The starting point for all tasks described in this chapter is the raw data,
+containing the exact data files acquired, with no changes or additions.
+Be mindful of how and where these files are stored
+as they cannot be re-created and nearly always contain confidential information.
+Raw data files must \textit{never} be edited directly.
+If the raw data contains confidential information,
+any tasks that precede de-identification must be carried on encrypted data.
+ 
+% Why cleaning is necessary
+Raw data files may be received in a variety of formats, 
+and are often not easy to handle using statistical software.
+The data cleaning tasks described in this chapter will prepare the raw data
+for quantitative analysis.
+To make this complex process less error-prone,
+we recommend that no changes be made to the content of data at this point
+(with one exception that will be carefully discussed).
+Correcting outliers, imputing values and creating derived indicators
+are all research decisions, and are therefore considered part of analysis (discussed in the next chapter).
+Therefore, the main output at this stage, the cleaned dataset,
+contains almost exactly same information as the raw data,
+but in a format that is easier to interact with.  
+
+\section{Identifying the identifier(s)}
+
+% Intro
+The very first step in creating statistical software-friendly data sets
+is understanding the data acquired,
+and translating this understanding into an intuitive format.
+This section discusses what steps may be needed to make sure that each row
+in a dataset represents one observation.
+This may seem trivial, but should not be taken for granted.
+Getting to such a format may be harder than expected,
+and the \textbf{unit of observation}\sidenote{\textbf{Unit of observation:}
+	the unit described by the data. \url{
+		https://dimewiki.worldbank.org/wiki/Unit_of_Observation}}
+ may be ambiguous in many raw datasets.
+But having data in a tidy format is the best way to manipulate it, 
+particularly aggregating and cross-referencing.
+You will need to do manipulate data as soon as it is received,
+so you can assure its quality.
+If you are conducting primary data collection,
+you can start coding the data tidying before the data is acquired,
+since you will know in advance in which format it will be received.
+
+\subsection{Unique identifiers}
+
+% Uniquely and fully identifying variable
+The first step in tidying data is to understand its unit of observation (what makes a row),
```

Comment:
```suggestion
The first step in tidying data is to understand its unit of observation,
```
I do not think we should say something that seem to put an equality sign between observation and row. A row is often an observation, but, especially in new data sources, an observation can be a time period or a file (ex and image).  

I know I just suggested adding row as an description of what unit of observation means above. I think it is a good way to explain the concept, but we should stay away from `observation==row` as it is less and less true in both direction in new data sources.

Reply:
I see. You're right. There's no good vocabulary here though, since we typically use "observation" to call a row. Maybe in this case we should say "The first step in tidying tabular data is to understand what unit is represented in each row"? I'm not sure it makes sense to talk about tidy data for non-tabular data.

Reply:
Do we say something about tidy data is something specific to tabular/relational datasets? If we caveat this whole section with that, then we can be more relaxed about statements like this.